k8s/backend-simple.yaml

```
@@ -0,0 +1,371 @@
+            console.log('📈 GET /api/v1/metrics/live-requests - returning mock requests');
+            res.json({
+              success: true,
+              data: mockLiveRequests,
+              timestamp: new Date().toISOString()
+            });
+          });
+          
+          // Request simulator endpoint
+          app.post('/api/v1/simulator/chat/completions', (req, res) => {
+            console.log('🔄 POST /api/v1/simulator/chat/completions - simulating request');
+            
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)